solar_system.rb

```diff
@@ -0,0 +1,208 @@
+
+# planet_names = [ "Mercury", "Venus", "Earth", "Mars", "Jupiter", "Pluto" ] # wave 1a
+
+mercury =
+  {
+    name: "Mercury",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+
+venus =
+  {
+    name: "Venus",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+
+  earth =
+  {
+    name: "Earth",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+
+  mars =
+  {
+    name: "Mars",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+
+  jupiter =
+  {
+    name: "Jupiter",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+
+  saturn =
+  {
+    name: "Saturn",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+
+  uranus =
+  {
+    name: "Uranus",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+
+  neptune =
+  {
+    name: "Neptune",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+# instead of strings for planets, modify initialize method to take a list of hashes where each planet is sent as a hash with at least 5 attributes
+planet_names = [mercury, venus, earth, mars, jupiter, saturn, uranus, neptune]
+#new_planet = "Hoth"
+hoth =
+{
+  name: "Hoth",
+  distance_from_sun: 100,
+  color: "blarb",
+  moons: false,
+  size: "large",
+  sustains_life: false
+}
+new_planet = hoth
+
+class SolarSystem
+
+  def initialize(planet_names)
+    @planets = planet_names
+    #puts "Initializing beep boop"
+  end
+
+  def add_planet(new_planet)
+    @new_planet = new_planet
+    @planets.push(new_planet)
```

Comment:
I don't think you need to save `new_planet` as a separate instance variable here, since you're already adding it to `@planets`.